### DISCUSSION TITLE
Is telescope ignore float transparency?

### DISCUSSION THREAD

**Discussion by EpsilonKu** [neovide/neovide#1816] [Q&A]
I'm using neovim from wsl. It render blur and transparency well on lazy.nvim float. But it doesn't work on telescope. I tried default nvim colors, so doesn't helped. 
Neovide config:
```lua
vim.opt.guifont = { "FiraCode NFM", "Iosevka SS09", "JuliaMono Medium", ":h11" }
vim.g.neovide_floating_blur_amount_x = 2.0
vim.g.neovide_floating_blur_amount_y = 2.0
vim.g.neovide_floating_blur = true

```
Also I runned with `neovide --multigrid --wsl` command.

**Comment by fredizzimo** [comment]
Sorry for the very late reply, but the telescope has its own blend configuration https://github.com/nvim-telescope/telescope.nvim/blob/a19770625aed49ad2a9f591a5e3946707f7359f6/doc/telescope.txt#L208

But I think it only works together with https://github.com/neovide/neovide/pull/1977

**Comment by EpsilonKu** [comment]
Yeah, currently I am using telescope own winblend. It works good on stable.
But it different color for borders. I guess it's more like telescope problem.

**Reply by fredizzimo** [reply]
Yes, there’s this issue https://github.com/nvim-telescope/telescope.nvim/issues/2603
